docs/application/dotnet/guides/app-management/widget-management.md

```diff
@@ -1,6 +1,6 @@
-# Widget Control
+# Widget Management
```

Comment:
@mihashco, Per below comment, please change the base branch from "Tizen 6.5" to "master" for this PR.
https://github.com/Samsung/tizen-docs/pull/1507#issuecomment-926521854

@annie-samsung, You are correct, this PR should have been raised on master branch. Thank you for pointing this out.